### ISSUE TITLE
Paths that include the package name causes conflicts

### ISSUE THREAD

**Issue by piepieonline** [bholloway/omit-tilde-webpack-plugin#2]
`WARNING in omit-tilde-webpack-plugin files should use ~ to refer to modules: change "spin.js" -> "~spin.js"`
where the path is: 
`App_Root/bower_components/spin.js/spin.js`

It would appear that the directory being 'spin.js' is causing issues, incorrectly suggesting that a tilde needs to be added.

**Comment by piepieonline** [comment]
Logging `candidate` at line 81:

```
{ 
  path: '/Applications/XAMPP/xamppfiles/htdocs/deployment-tool/bower_components/spin.js',
  request: './spin.js',
  query: null,
  directory: false 
}        
```

**Comment by bholloway** [comment]
Thanks @piepieonline 

Could you please try installing `omit-tilde-webpack-plugin@0.1.0` and let me know if it solves the problem.

**Comment by bholloway** [comment]
closed by #6
